### ISSUE TITLE
Show more experiment configuration on chaos dashboard

### ISSUE THREAD

**Issue by cwen0** [chaos-mesh/chaos-mesh#1319]
## Feature Request

**Is your feature request related to a problem? Please describe:**

**Describe the feature you'd like:**

Show more experiment configuration on chaos dashboard, such as selectors and target

![image](https://user-images.githubusercontent.com/22956341/102861520-c6f37c00-446a-11eb-8734-ca0872f82f52.png)

**Comment by fewdan** [comment]
Now these fields should not exist in the db, so for compatibility, should query the corresponding information from kubernetes ?

**Comment by victoriacheng15** [comment]
👋 @g1eny0ung

I noticed that PR #1977 introduced `ObjectConfiguration` (`ui/app/src/components/ObjectConfiguration/index.tsx`) to display experiment configurations such as selectors and targets on the dashboard.

Does PR #1977 address what was requested in this issue?

**Comment by g1eny0ung** [comment]
Yep. This is still an issue that was forgotten to be closed. I'm closing it now. Thanks for the reminder!
